eth/vm/computation.py

```diff
@@ -258,6 +287,9 @@ def memory_read_bytes(self, start_position: int, size: int) -> bytes:
         """
         return self._memory.read_bytes(start_position, size)
 
+    def dump_memory(self) -> bytes:
+        return bytearray(self._memory)
```

Comment:
Maybe use the new `read_bytes` like:

```suggestion
        return self._memory.read_bytes()
```